src/fielddef.ts

```diff
@@ -48,6 +48,15 @@ export interface ValueDef {
  */
 export type ChannelDefWithCondition<F extends FieldDef<any>> = FieldDefWithCondition<F> | ValueDefWithCondition<F>;
 
+/**
+ * A ValueDef with Condition<ValueDef | FieldDef> where either the conition or the value are optional.
+ * {
+ *   condition: {field: ...} | {value: ...},
+ *   value: ...,
+ * }
+ */
+export type ValueDefWithCondition<F extends FieldDef<any>> = ValueDefWithOptionalCondition<F> | ConditionOnlyDef<F>;
```

Comment:
Technically, `ConditionOnlyDef` isn't a ValueDef, but I think this is good for the docs since the docs won't include `ConditionOnlyDef`, which is a bit too complicated to include.